opentracing-impl-java8/src/main/java/io/opentracing/AbstractSpan.java

```diff
@@ -127,18 +123,46 @@ public String getBaggageItem(String key) {
     }
 
     @Override
-    public final Span log(String message, /* @Nullable */ Object payload) {
+    public final Span log(String event) {
+        return log(System.nanoTime() / 1000, event);
```

Comment:
@yurishkuro I replaced `nano/1000` with `millis*1000`... not really relevant to this PR, but such a small change I figured I'd include it.

(In AbstractSpan I was able to use `Instant` which obviates the problem... though this points to additional problems with the whole `AbstractSpan` concept. I would really like to get that whole artifact out of this particular github repo, but that's its own discussion.)